www/class/centreon-clapi/Validator/RtValidator.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * Copyright 2005 - 2022 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+namespace CentreonClapi\Validator;
+
+use CentreonClapi\CentreonHost;
+use CentreonClapi\CentreonRtAcknowledgement;
+use CentreonClapi\CentreonRtDowntime;
+use CentreonClapi\CentreonService;
+
+/**
+ * This class is used to validate inputs for RT clapi endpoints.
+ *
+ * @see CentreonRtAcknowledgement
+ * @see CentreonRtDowntime
+ */
+class RtValidator
+{
+    /**
+     * @var CentreonHost
+     */
+    private $hostObject;
+
+    /**
+     * @var CentreonService
+     */
+    private $serviceObject;
+
+    public function __construct(CentreonHost $hostObject, CentreonService $serviceObject)
+    {
+        $this->hostObject = $hostObject;
+        $this->serviceObject = $serviceObject;
+    }
+
+    /**
+     * Mysql perform an insensitive case search. We want to enforce the strictness here.
```

Comment:
```suggestion
     * Data storage perform an insensitive case search. We want to enforce the strictness here.
```
we should avoid to mentions the type of datastorage